app/models/setting/bootdisk.rb

```diff
@@ -13,6 +15,7 @@ def self.load_defaults
         self.set('bootdisk_ipxe_dir', N_('Path to directory containing iPXE images'), ipxe, N_('iPXE directory')),
         self.set('bootdisk_isolinux_dir', N_('Path to directory containing isolinux images'), isolinux, N_('ISOLINUX directory')),
         self.set('bootdisk_syslinux_dir', N_('Path to directory containing syslinux images'), syslinux, N_('SYSLINUX directory')),
+        self.set('bootdisk_efiboot_dir', N_('Path to directory containing efiboot.img and grubx64.efi (leave empty for BIOS only)'), '', N_('EFI directory')),
```

Comment:
1. Missing from README.
1. Why default to an empty path? Isn't this known for most OSes?